File: app/src/main/java/com/sanketvagadiya/ui/Profile/ProfileViewModel.java
package com.sanketvagadiya.ui.Profile;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class ProfileViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public ProfileViewModel(){
        mText = new MutableLiveData<>();
        mText.setValue("this is profile");

    }

    public LiveData<String> getText(){return mText;}
}

File: app/src/main/java/com/sanketvagadiya/ui/Profile/ProfileViewModel.kt
package com.sanketvagadiya.ui.Profile

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class ProfileViewModel : ViewModel() {

    private val mText: MutableLiveData<String>

    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "this is profile"

    }
}
